src/cli/repo/s3.go

```diff
@@ -128,6 +129,13 @@ func initS3Cmd(cmd *cobra.Command, args []string) error {
 		return Only(ctx, errors.Wrap(err, "Retrieving s3 configuration"))
 	}
 
+	if strings.HasPrefix(s3Cfg.Endpoint, "http://") || strings.HasPrefix(s3Cfg.Endpoint, "https://") {
+		invalidEndpointErr := "endpoint doesn't support specifying protocol. " +
+			"pass --disable-tls flag to use http:// instead of default https://"
+
+		return Only(ctx, errors.New(invalidEndpointErr))
```

Comment:
IIUC, `Only` is supposed to print just the error without a help message, but somehow it seems to be doing this. It is unrelated to this PR, so don't have  to address in this one, but just mentioning to keep track.

Reply:
noted!!